modules/check_in/spec/services/v2/lorota/service_spec.rb

```diff
@@ -280,6 +280,10 @@
 
   before do
     allow(Rails).to receive(:cache).and_return(memory_store)
+    allow(Flipper).to receive(:enabled?).with('check_in_experience_504_error_mapping_enabled')
```

Comment:
this no longer exists, you can probably remove it

Reply:
Fixed in [61b8213](https://github.com/department-of-veterans-affairs/vets-api/commit/61b8213df055c53ee4e72a4107a4b5553956b97c)